docs/messaging-implementation.md

```diff
@@ -136,7 +136,7 @@ Example widget configuration
     <value>["result", 0, "message"]</value>
 </portlet-preference>
 <portlet-preference>
-    <name>widgetURL</name>
+    <name>widgetExternalMessageLearnMoreUrl</name>
     <value>["learnMoreUrl"]</value>
```

Comment:
I'm left confused about whether I should expect the `<value>` of `widgetExternalMessageLearnMoreUrl` to be, well, a URL? Like, is a valid example value literally `["learnMoreUrl"]`?